schema.sql

```diff
@@ -1 +1,8 @@
-/* Database schema to keep the structure of entire database. */
\ No newline at end of file
+CREATE TABLE animals (
+    id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [ OPTIONAL]
 You can make the id NOT NULL and autoincrement: `id int NOT NULL IDENTITY(1,1)`